### PR TITLE
ja: Make docs/concepts/services-networking/ingress-controllers.md follow v1.19 of the original text

### LINKED ISSUE #26889: ja: Make docs/concepts/services-networking/ingress-controllers.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/concepts/services-networking/ingress-controllers.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/concepts/services-networking/ingress-controllers.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/services-networking/ingress-controllers.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/services-networking/ingress-controllers.md

diff between translated and v1.19
https://gist.github.com/5329e3577bcc47787d4ebbdaa3188e2d

**Note**
Currently, we use **dev-1.19-ja.1** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by shuuji3**
✅  Resolves #26889.

### CODE REVIEW

**Comment by shuuji3** [comment]
`i18n/ja.toml`を変更したせいか、enの人がサジェストされてしまっているようです。
/assign inductor

File: content/ja/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -11,47 +11,42 @@ Ingressリソースが動作するためには、クラスターでIngressコン
...
-* [NGINX, Inc.](https://www.nginx.com/)は[NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx/kubernetes-ingress-controller)のサポートと保守を提供しています。
+{{% thirdparty-content %}}
+
+* [AKS Application Gateway Ingress Controller](https://github.com/Azure/application-gateway-kubernetes-ingress)は、[Azure Application Gateway](https://docs.microsoft.com/azure/application-gateway/overview)を設定するIngressコントローラーです。
+* [Ambassador](https://www.getambassador.io/) API Gatewayは[Envoy](https://www.envoyproxy.io)ベースのIngressコントローラーです。
+* [Citrix ingress controller](https://github.com/citrix/citrix-k8s-ingress-controller#readme)は、Citrix Application Delivery Controllerで動作します。
+* [Contour](https://projectcontour.io/)は、[Envoy](https://www.envoyproxy.io/)ベースのIngressコントローラーです。
+* F5 BIG-IPの[Container Ingress Services for Kubernetes](https://clouddocs.f5.com/containers/latest/userguide/kubernetes/)はF5 BIG-IPの仮想サーバー上でIngressの設定を可能にします。
+* [Gloo](https://gloo.solo.io)は[Envoy](https://www.envoyproxy.io)をベースにしたオープンソースのIngressコントローラーです。
```
Comment (atoato88) [inline content/ja/docs/concepts/services-networking/ingress-controllers.md:27]: PR codeset does not include a sentence `API Gateway機能を提供しています`, is it OK?
Original sentence is:
`Gloo is an open-source ingress controller based on Envoy, which offers API gateway functionality.`

Comment (shuuji3) [reply content/ja/docs/concepts/services-networking/ingress-controllers.md]: Sorry, it's my error. I added your suggestion.

Comment (atoato88) [reply content/ja/docs/concepts/services-networking/ingress-controllers.md]: Thank you to update! :smile:

**Pull request merged**